packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Changelog
 
 ## Unreleased
+
 * Changed
+  * Updated example for `terra-popup`.
+  * Updated examples and tests for `terra-worklist-data-grid` to remove row height.
```

Comment:
two same entries been added here. Please remove one
`Updated examples and tests for `terra-worklist-data-grid` to remove row height`

Reply:
Resolved "https://github.com/cerner/terra-framework/pull/1660/commits/ff6fdf4bec6b90ceae1bc894cffb9aaebe5ff2ed"